### PR TITLE
 week4 homework

### PR DESCRIPTION
**Pull request by Gohar33**

### CODE REVIEW

File: javascript/javascript1/week4/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        if (number1 && number2);
+        switch (operator) {
+            case "-":
+                return number1 - number2;
+            case "+":
+                return number1 + number2;
+            case "*":
+                return number1 * number2;
+            case "/":
+                return number1 / number2;
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/index.js:55]: What if the operator is not known? Can you add a `default` case to this switch statement to handle any math operator that is not known? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

File: javascript/javascript1/week4/index.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            (function () {
+                console.log("Timer done")
+            }, 240000);
+        return "Timer is set";
+    } else if (command.startsWith("What's")) {
+        let numbers = command.split(" ");
+        let number1 = parseInt(numbers[1]);
+        let number2 = parseInt(numbers[3])
+        let operator = numbers[2];
+        if (number1 && number2);
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/index.js:46]: Does this line do anything? My guess is that it doesn't.

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week4/index.js (hunk 2 above)
Comment (Gohar33) [reply javascript/javascript1/week4/index.js:46]: Thanks Cameron, yes, you're right, I forgot to delete it while testing few options.

File: javascript/javascript1/week4/index.js (hunk 1 above)
Comment (Gohar33) [reply javascript/javascript1/week4/index.js:55]: But if the operator is unknown how can I know which kind of action I should assign to it? I can add a default but I don't know which mathematical calculations I should assign?

Comment (cameronjdavis) [reply javascript/javascript1/week4/index.js:54]: True, you don't know how to handle an unknown operator but the user could enter an unknown operator. You should add a `default` option to tell the user that you can't handle the unknown operator.

Comment (Gohar33) [reply javascript/javascript1/week4/index.js:54]: Oh great, I did exactly that in the feedback implementation, just added default and returned "Error". Thanks for the feedback and for pushing us to think about more options, that's very informative and helpful :)

**Pull request merged**